### ISSUE TITLE
Cannot reproduce creating the hash

### ISSUE THREAD

**Issue by StephanMeijer** [wordproof/timestamp-standard#2]
See https://github.com/wordproof/timestamp-standard/blob/master/ArticleTimestamp.md#creating-the-hash

Documentation says the following content will create the following hash: `8C9453EF2C0C6F2F94104D16FF0A8105BC6C9793D926457839D8BCADE0888342`

```json
{
  "type": "ArticleTimestamp",
  "version": "0.2.0",
  "title": "My Blog Title",
  "content": "Lorem Ipsum",
  "date": "2019-07-08T08:38:23+0000"
}
```

My results are as following, using Python 3:

```
Python 3.7.5 (default, Apr 19 2020, 20:18:17) 
[GCC 9.2.1 20191008] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import hashlib
>>> def hash(str):
...   return hashlib.sha256(str).hexdigest()
... 
>>> hash(b"""{
...   "type": "ArticleTimestamp",
...   "version": "0.2.0",
...   "title": "My Blog Title",
...   "content": "Lorem Ipsum",
...   "date": "2019-07-08T08:38:23+0000"
... }""")
'0cce4df0dfdd4ad6cc9fd67baac8f1fd796d746d7daa0bf86150828c75d5fdb5'
>>> hash(b"""{"type":"ArticleTimestamp","version":"0.2.0","title":"My Blog Title","content":"Lorem Ipsum","date":"2019-07-08T08:38:23+0000"}""")
'203bc0d768c700ccd26944a76a4d568299813dfbf631414f4b22db3b9c787b7d'
```

PHP:
```php
<?php

$articleJSON = <<<JSON
{
  "type": "ArticleTimestamp",
  "version": "0.2.0",
  "title": "My Blog Title",
  "content": "Lorem Ipsum",
  "date": "2019-07-08T08:38:23+0000"
}
JSON;

echo hash('sha256', $articleJSON);
echo "\n";
echo hash("sha256", '{"type":"ArticleTimestamp","version":"0.2.0","title":"My Blog Title","content":"Lorem Ipsum","date":"2019-07-08T08:38:23+0000"}');
```

Output:
```
23bd0578abed1585b05337abe164f36b797a422d7b1fea2f266d80a5b5085072
203bc0d768c700ccd26944a76a4d568299813dfbf631414f4b22db3b9c787b7d
```

Could you point me where I go wrong?

**Comment by marijnbent** [comment]
Hi Stephan,

Thanks for opening an issue. As you have demonstrated, the hash in the example is wrong and should have been updated. The correct hash is `203bc0d768c700ccd26944a76a4d568299813dfbf631414f4b22db3b9c787b7d`, the outcome of the minified JSON object which you used in your Python and second PHP example. 

I've updated ArticleTimestamp.md with the correct hash and the requirement to use a minified JSON object.
